src/Microsoft.AspNet.Mvc.Core/DefaultControllerActionArgumentBinder.cs

```diff
@@ -100,6 +104,14 @@ private void UpdateParameterMetadata(ModelMetadata metadata, IBinderMetadata bin
                     modelBindingContext.IsModelSet)
                 {
                     arguments[parameter.PropertyName] = modelBindingContext.Model;
+                    var validationContext = new ModelValidationContext(
+                    _modelMetadataProvider,
```

Comment:
mm, sure dropping this we can directly depend on metadata.properties to navigate. Wondering what is your reason for dropping this.